RealmSwift/Tests/ObjectCreationTests.swift

```diff
@@ -441,7 +441,7 @@ class ObjectCreationTests: TestCase {
         Realm().commitWrite()
         switch type {
             case .Bool:     return ["invalid", 2 as Int, 1.1 as Float, 11.1 as Double]
-            case .Int:      return ["invalid", true, false, 1.1 as Float, 11.1 as Double]
```

Comment:
Shouldn't we now add `true` and `false` to `validValuesForSwiftObjectType` for Int?